src/validators/mosaic.validator.ts

```diff
@@ -46,3 +47,12 @@ export class MosaicsValidator implements Validator<string> {
         });
     }
 }
+
+export class MosaicRestrictionTypeValidator implements Validator<string> {
```

Comment:
This file https://github.com/nemtech/nem2-cli/blob/13dd3e7eccb4c1b5c59732506b5b857b9cacdac8/src/validators/restrictionType.validator.ts#L38

already contains the ``MosaicRestrictionTypeValidator``